### DISCUSSION TITLE
github repo reaached the maximum number of statuses

### DISCUSSION THREAD

**Discussion by vijayscsa** [community/community#30812] [Other Feature Feedback, Questions, & Ideas]
Hi,

My pipeline keeps fail eveytime with the below error message
**_Could not update commit status. Message: {"message":"Validation Failed","errors":[{"resource":"Status","code":"custom","message":"This SHA and context has reached the maximum number of statuses."}],"documentation_url":"https://docs.github.com/enterprise/3.3/rest/reference/repos#create-a-commit-status"}_**

Request your help, what to do how shall increase the limit or reduce the existing connections by clearing sessions etc.  Atleast some work aounds will be helpful.  Thanks

**Comment by fredrikmg** [comment]
Got the exact same problem.
